typings/index.d.ts

```diff
@@ -1422,6 +1423,7 @@ declare module 'discord.js' {
   export class SnowflakeUtil {
     public static deconstruct(snowflake: Snowflake): DeconstructedSnowflake;
     public static generate(timestamp?: number | Date): Snowflake;
+    public static maybeSnowflake(string: string): string is Snowflake;
```

Comment:
Literally `string is string`...
But can be valid after https://github.com/discordjs/discord.js/pull/5157.

Reply:
I suggest `value`:
```suggestion
    public static maybeSnowflake(value: string): value is Snowflake;
```

Reply:
I mean, I don't really mind changing it but I'd argue this is an improvement. `string` describes exactly what that argument is: any given string. If anything, `value` is even more broad than `string` is. I do agree that there are cases when using `string` as an argument name is no good, but I reckon this is not one of them.

Give me an objective reason `string` is bad and why you think other names are better and I will change it straight away. Subjective thoughts and wishes should have no influence on code.

Reply:
The `string` argument name, is very confusely with the type.
Moreover, `string` is not really good for an argument name, because it does not describe the argument. `string` can tell anything. `value` is more precise, less confusing and better.

Reply:
I agree with the first part: it can be confused with the type. I will update it in regards to that.

Though your second argument really makes no sense to me. `value` is more precise in what way exactly? `string` tells you it's a string, `value` can be anything 😄

Reply:
Not really, because it's typed as a `string`, it doesn't just accept any *value*.

As far as that typing is concerned, it reads as a "value of type string", which sounds better than "string of type string".

Reply:
For me, it's more precise than `string`. There are maybe better names than `value` (who is better than `string`). `possibleSnowflake` (no ?) ?

Reply:
Updated to `value`